primitives/npos-elections/src/lib.rs

```diff
@@ -650,12 +650,18 @@ pub(crate) fn setup_inputs<AccountId: IdentifierT>(
 				);
 			} // else {} would be wrong votes. We don't really care about it.
 		}
-		Voter {
-			who,
-			edges: edges,
-			budget: voter_stake.into(),
-			load: Rational128::zero(),
+		if edges.is_empty() {
+			None
+		}
+		else {
```

Comment:
I think this is the standard format in substrate
```suggestion
		} else {
```